mutiny-core/src/lib.rs

```diff
@@ -487,7 +544,29 @@ impl<S: MutinyStorage> MutinyWallet<S> {
 
         Ok(mw)
     }
+}
 
+#[derive(Clone)]
+/// MutinyWallet is the main entry point for the library.
+/// It contains the NodeManager, which is the main interface to manage the
+/// bitcoin and the lightning functionality.
+pub struct MutinyWallet<S: MutinyStorage> {
```

Comment:
nit: can we have the comment on top of the `#[derive(..)]` this always bothered me